No GUdev on OpenBSD (XXX add HAVE_GUDEV to meson_options.txt)

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -190,7 +190,7 @@ libxml_dep = dependency('libxml-2.0')
 pulse_dep = dependency('libpulse', version: pulse_req_version)
 pulse_mainloop_dep = dependency('libpulse-mainloop-glib', version: pulse_req_version)
 upower_glib_dep = dependency('upower-glib', version: '>= 0.99.8')
-gudev_dep = dependency('gudev-1.0', version: '>= 232')
+gudev_dep = []
 x11_dep = dependency('x11', version: '>= 1.8')
 xi_dep = dependency('xi', version: '>= 1.2')
 epoxy_dep = dependency('epoxy')
